src/js/media-chrome-menu-button.js

```diff
@@ -62,12 +63,9 @@ class MediaChromeMenuButton extends MediaChromeButton {
     }
   }
 
-  mediaSetCallback(media) {}
+  mediaSetCallback(media) { }
 }
 
-if (!window.customElements.get('media-chrome-menu-button')) {
-  window.customElements.define('media-chrome-menu-button', MediaChromeMenuButton);
-  window.MediaChrome = MediaChromeMenuButton;
```

Comment:
This line is one of the lines that initiated the conversion. There are a few components that register `window.{COMPONENT_NAME}` incorrectly.

Reply:
Ha! Oops.